arviz/stats/stats_utils.py

```diff
@@ -439,7 +439,7 @@ def get_log_likelihood(idata, var_name=None):
         return log_likelihood
 
 
-BASE_FMT = """Computed from {{n_samples}} by {{n_points}} log-likelihood matrix
+BASE_FMT = """Computed from {{n_samples}} samples and {{n_points}} points log-likelihood matrix.
```

Comment:
```suggestion
BASE_FMT = """Computed from {{n_samples}} posterior samples and \
{{n_points}} observations log-likelihood matrix.
```

I find points ambiguous and not much of an improvement over saying nothing. The `\` will allow breaking the line in the source code (therefore keeping lines below 100 characters) while keeping the printed message all in the same line.

Reply:
One other thing that also works is using `(`


    string1 = "abcd"
    string2 = (
        "a"
        "b"
        "c"
        "d"
    )
    string1 == string2